test/parallel/test-http-destroyed-socket-write2.js

```diff
@@ -40,11 +44,9 @@ server.listen(0, function() {
     method: 'POST'
   });
 
-  function write() {
-    req.write('hello', function() {
-      setImmediate(write);
-    });
-  }
+  server.once(kResponseDestroyed, common.mustCall(function() {
+    req.write('hello');
```

Comment:
Is there a need for `.mustCall` here?

Reply:
Yes, otherwise there is no guarantee that the listener of the `'request'` event added on line 33 is called.

Reply:
Anyway we can use `common.mustCall()` there (line 33) if it is easier to grok.

Reply:
I think I misunderstood. Do you mean adding a callback to this `req.write()`? If so, I think it is not needed. It's out of the scope of the test.